Initial documentation

NicolasT opened the pull request:
Add some initial documentation, based on Sphinx.

NicolasT commented:
Would :heart: approval from @thomasdanan once completed the Quickstart guide.

LaureVergeron commented on README.md:
```diff
@@ -1,13 +1,22 @@
 
-## Production Config
+## Documentation
+The project documentation is available at https://metal-k8s.readthedocs.io.
```
Shouldn't this be a link?

NicolasT replied:
It is.

LaureVergeron commented on README.md:
```diff
@@ -1,13 +1,22 @@
+## Hardware Requirements
+
+### Test config
+If you want  to test metal-k8s you just need one simple VM running CentOS 7.4.
```
double space

NicolasT replied:
Fixed

LaureVergeron commented on README.md:
```diff
@@ -17,56 +26,5 @@ For production-ready environment, you need:
-- https://kubernetes.io/docs/tasks/tools/install-kubectl/
-- https://docs.helm.sh/using_helm/#installing-helm
+## Software Requirements
+To get started, all you need is `python` and `make` on a Unix-like system.
```
which versions?

NicolasT replied:
Doesn't matter, which is exactly the point.

NicolasT replied:
@wabernatScality Feedback welcome at any point in time as far as I'm concerned :smile: Though maybe a bit too early now, I'll add more content first. Thanks for the :+1:!

NicolasT commented:
Change since previous reviews: addition of the *Cluster Services* document, see https://github.com/scality/metal-k8s/blob/docs/docs/architecture/cluster-services.rst

NicolasT commented:
Agree we should try to get patterns set up in Kibana by default, though that's out of scope for this PR :) I guess something should be feasible through a Helm `init job`.
